code/modules/reagents/Chemistry-Reagents.dm

```diff
@@ -3370,7 +3329,6 @@
 		return 1
 
 	M.nutrition += nutriment_factor
-	M.bodytemperature += 10 * TEMPERATURE_DAMAGE_COEFFICIENT
```

Comment:
Hell Ramen can no longer be used to burn someone to death? 👎 

That's the whole point of Hell Ramen. If you take that out it's totally identical to Hot Ramen.